client/my-sites/comments/main.jsx

```diff
@@ -47,12 +52,33 @@ export class CommentsManagement extends Component {
 			translate,
 		} = this.props;
 
+		const jetpackUpdateLink = `/plugins/jetpack/${ this.props.siteSlug }`;
+
+		const showJetpackUpdateScreen =
+			isJetpack &&
+			! jetpackCommentsManagementSupported &&
+			config.isEnabled( 'comments/management/jetpack-5.5' );
+
 		return (
 			<Main className="comments" wideLayout>
 				<PageViewTracker path="/comments/:status/:site" title="Comments" />
 				<DocumentHead title={ translate( 'Comments' ) } />
-				<SidebarNavigation />
-				{ showPermissionError && (
+				{ showJetpackUpdateScreen && (
+					<EmptyContent
+						title={ preventWidows( translate( "Looking to manage this site's comments?" ) ) }
+						line={ preventWidows(
+							translate( 'We need you to update to the latest version of Jetpack' )
+						) }
+						illustration="/calypso/images/illustrations/illustration-404.svg"
+						action={ translate( 'Update Jetpack' ) }
+						actionURL={ jetpackUpdateLink }
+						secondaryAction={ translate( 'Enable autoupdates' ) }
+						secondaryActionURL={ jetpackUpdateLink }
```

Comment:
The fact that both buttons point to the same URL is a bit weird to me.

If we can't automatically trigger those actions (`update`, `enable autoupdate`), maybe it's better to have a button that explains both options?

> Update Jetpack or enable autoupdates

Though, it's a mouthful and looks quite awful...

Any opinions @MichaelArestad?

Reply:
If we are going to change anything, I'm more inclined to just remove the autoupdates button for now, but let's wait for other folks to chime in too. :)

Reply:
Yeah, I mean, it's a very, very, very minor nitpick.

Question is: is the point of the Autoupdates button just to remind users that there is an autoupdate feature?
In this case, couldn't we achieve it with a simple message, instead of a button?

Reply:
Yep, I'd say just get rid of the autoupdates button ("why are they talking about autoupdates, I thought I needed a new version?..").

Reply:
cc @MichaelArestad having a secondary call to action is a bit confusing now that I look at it more closely. It feels like it could work as a checkbox option, but we could perhaps iterate on that in the future.

Reply:
We want to give them an option to enable autoupdates so they never have to see this screen again. It's not necessary, but would be nice if we could work it in.

Reply:
Both buttons should just initiation the action. "Update" should start updating. "Enable autoupdates" should turn on autoupdates for the plugin.

Reply:
@vindl are we not able to do this in one action? We could perhaps just add the plugin update action to the data-layer if it's not available yet.

Reply:
> We could perhaps just add the plugin update action to the data-layer if it's not available yet.

@gwwar I don't think that that's possible given that Plugins store in not reduxified yet?

Reply:
@vindl Is the `updatePlugin` action not usable here? https://github.com/Automattic/wp-calypso/blob/master/client/state/plugins/installed/actions.js#L119

Reply:
Thanks! I'll check it out.

Reply:
Removed secondary autoupdate action and tied button click to `updatePlugin` action in 5676b5222406d81fea2771233224456b58406758.